Documentation/components/drivers/special/sensors/l86xxx.rst

```diff
@@ -0,0 +1,149 @@
+The latter is used to send proprietary 'PMTK' commands which are documented further below.
+
+**Activate**
+
+There are 4 modes that the L86-XXX GNSS modules can be in. Those are "Full On Mode", "Standby Mode", "Backup Mode", "Periodic Mode" and, "AlwaysLocateTM Mode".
+Calling ``orb_activate`` with ``enable`` set to false will enter the module into "Standby Mode". 
+In "Standby Mode", the module doesn't output any NMEA messages but the internal core and I/O power domain are still active.
+
+The module can be re-enabled by calling ``orb_activate`` with ``enable`` set to true, which will hot start the module OR by
+sending any 'PMTK' command.
+
+**Set interval**
+
+The L86-XXX GNSS modules support interval rates from 1Hz to 10Hz (100ms - 10000ms). When using ``orb_set_interval``, be aware that
+increasing the interval of the module may also require and increase in baud rate. An example of how this is performed can be found in
+source code of this driver in the register function.
+
+Any interval rate outside of the supported range will result in a failed call to this function.
+
+**Control**
+
+The ``orb_ioctl`` interface allows one to send proprietary 'PMTK' commands to the L86-XXX GNSS module. It effectively works
+as a wrapper for the command framework outlined by Quectel. The return value of calls to ``orb_ioctl`` mimic the return values
+from command acknowledgements from the GNSS module:
+
+* 0: Invalid packet
+* 1: Unsupported packet type
+* 2: Valid packet, but action failed
+* 3: Valid packet, action succeeded  
+
+The supported commands are their arguments are listed below.
+
+``CMD_HOT_START``
+-----------------
+Used to "Hot start" the GNSS module. There is no acknowledgement for this command.
+
+.. code-block:: c
+
+   orb_ioctl(sensor, CMD_HOT_START);
+
+``CMD_WARM_START``
+------------------
+Used to "Warm start" the GNSS module. There is no acknowledgement for this command.
+
+.. code-block:: c
+
+   orb_ioctl(sensor, CMD_WARM_START);
+
+``CMD_COLD_START``
+------------------
+Used to "Cold start" the GNSS module. There is no acknowledgement for this command.
+
+.. code-block:: c
+
+   orb_ioctl(sensor, CMD_COLD_START);
+
+``CMD_FULL_COLD_START``
+-----------------------
+Used to "Full cold start" the GNSS module. There is no acknowledgement for this command.
+
+.. code-block:: c
+
+   orb_ioctl(sensor, CMD_FULL_COLD_START);
+
+``CMD_STANDBY_MODE``
+--------------------
+Used to enter "Standby Mode". This command must be called with ``0`` as an argument. The acknowledgement for this command is handled by the ``ioctl`` call.
+
+.. code-block:: c
+
+   orb_ioctl(sensor, CMD_STANDBY_MODE, 0);
+
+``SET_POS_FIX``
+---------------
+Used to modify the position fix interval of the GNSS module. The argument is an integer between 100 and 10000, default value is 1000. The acknowledgement for this command is handled by the ``ioctl`` call.
```

Comment:
No need to mention for any of these that the acknowledgement is handled by the `ioctl` call. The user won't care (and probably doesn't know how the GPS works under the hood), so this may confuse them.